packages/block-library/src/navigation/index.php

```diff
@@ -1520,6 +1520,14 @@ function block_core_navigation_update_ignore_hooked_blocks_meta( $post ) {
 		return $post;
 	}
 
+	/**
+	 * Skip meta generation when consumers intentionally update specific Navigation fields
+	 * and omit the content update.
+	 */
+	if ( ! isset( $post->post_content ) ) {
```

Comment:
Noting that this is `stdClass` because it’s hooked to https://developer.wordpress.org/reference/hooks/rest_pre_insert_this-post_type/.